### PR TITLE
Add commit value to onnx file and clean up some code

### PR DESCRIPTION
**Pull request by hwangdeyu**
- Add commit value to onnx file to trace precise commit version
- Remove `__future__` imports that are no-ops on latest Python. 
- Replace some single quotes to double quotes.
- Add License automatically in generating `version.py`.
- Change onnx link master to main

Signed-off-by: Deyu Huang <deyhuang@microsoft.com>
Co-authored-by: Guenther Schmuelling <guschmue@microsoft.com>

### CODE REVIEW

File: tf2onnx/graph.py [hunk 1]
```diff
@@ -1189,9 +1189,10 @@ def make_model(self, graph_doc, optimize=False, graph_name="tf2onnx", external_t
         graph = self.make_graph(graph_doc, graph_name, external_tensor_storage)
 
         if "producer_name" not in kwargs:
-            kwargs = {"producer_name": "tf2onnx",
-                      "producer_version": __version__}
-
+            kwargs = {
+                "producer_name": "tf2onnx",
+                "producer_version": __version__ + " " + git_version[:6]
```
Comment (hwangdeyu) [inline tf2onnx/graph.py:1194]: Get the first 6 digits of one commit.
Use `git show 6003d4` to check it.

Comment (hwangdeyu) [inline tf2onnx/graph.py:1194]: Get the first 6 digits of one commit.
Use `git show 6003d4` to check it.

File: tf2onnx/tfonnx.py [hunk 2]
```diff
@@ -411,7 +411,7 @@ def process_tf_graph(tf_graph, continue_on_error=False, verbose=False, target=No
 
     opset = utils.find_opset(opset)
     logger.info("Using tensorflow=%s, onnx=%s, tf2onnx=%s/%s",
-                get_tf_version(), utils.get_onnx_version(), tf2onnx.__version__, tf2onnx.version.git_version[:6])
+                get_tf_version(), utils.get_onnx_version(), tf2onnx.__version__, tf2onnx.git_version[:6])
```
Comment (fatcat-z) [inline tf2onnx/tfonnx.py:414]: Please revert it. Thanks.

File: tools/aggregate-patterns.py [hunk 3]
```diff
@@ -133,7 +133,7 @@ def get_args():
     parser.add_argument("--fold", action="store_true", help="fold smaller sub graphs")
     parser.add_argument("--max-nodes", type=int, default=6, help="number of max nodes in a pattern")
     parser.add_argument("--min-frequency", type=int, default=2, help="show patterns number seen at least this")
-    parser.add_argument("infile", nargs="*", help='event files')
+    parser.add_argument("infile", nargs="*", help="event files")
```
Comment (fatcat-z) [inline tools/aggregate-patterns.py:136]: Do we need to change it to "--infile"?

**Review by fatcat-z — APPROVED** [review]
LGTM! thanks!

**Pull request merged**